scss/_patterns_side-navigation.scss

```diff
@@ -216,44 +247,89 @@
     }
   }
 
-  .p-side-navigation__link {
-    @include vf-focus;
+  .p-side-navigation__status {
+    margin-left: auto;
+    padding-left: $spv-inner--small;
+  }
 
-    // vf-highlight-bar is rendered above focus outline
-    // so we need to hide it on focus
-    &:focus::before {
-      display: none;
+  // Styles for markup in Discourse docs variant
+  .p-side-navigation--discourse {
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      @extend %side-navigation__text;
+
+      // reset heading styles
+      font-size: map-get($base-font-sizes, base);
+      font-weight: $font-weight-bold;
+      line-height: 1.5rem;
+      margin: 0;
```

Comment:
How about this? Although, I like your simplicity.
```scss
@extend %bold;
@extend %default-text;
```

Reply:
Oh, wasn't aware we have those, will give it a go.

Reply:
Thanks for suggestion, I still had to reset margin and font size separately, but at least line height and font-weight is not hardcoded here now.